pulp_ansible/app/models.py

```diff
@@ -417,3 +417,44 @@ class AnsibleDistribution(Distribution):
 
     class Meta:
         default_related_name = "%(app_label)s_%(model_name)s"
+
+
+class RepositoryCollectionVersionIndex(models.Model):
```

Comment:
I think it would be helpful if the model name says explicitly that this is a `View` and not a table.

Because then we know that we can't call `objects.create` for example and other `CUD` methods.

Maybe `RepositoryCollectionVersionView`